### ISSUE TITLE
Adapt disabled test 'arg' availability in libraries from PUC-Rio Lua 5.1 test suite in main.lua

### ISSUE THREAD

**Issue by Buristan** [tarantool/tarantool#5686]
This ticket is a logical continuation of gh-4473.

If run  PUC-Rio Lua 5.1 test suite by LuaJIT (HEAD at the moment: [`1672bdc`](https://github.com/tarantool/luajit/commit/1672bdc0ffbee9f32fadb6943909d6af7eb9a7b1)) (not Tarantool) without any applied patches it fails with the following error:
```
/home/burii/builds_workspace/luajit/gh-4473-adopt-luavela-test-suites/src/luajit: /tmp/lua_I071Cz:1: assertion failed!
stack traceback:
        [C]: in function 'assert'
        /tmp/lua_I071Cz:1: in main chunk
        [C]: at 0x563e8a6affae
        [C]: at 0x563e8a67df96
/home/burii/builds_workspace/luajit/gh-4473-adopt-luavela-test-suites/src/luajit: main.lua:63: assertion failed!
stack traceback:
        [C]: in function 'assert'
        main.lua:63: in function 'RUN'
        main.lua:97: in function 'dofile'
        ...adopt-luavela-test-suites/test/PUC-Lua-5.1-tests/all.lua:69: in main chunk
        [C]: at 0x55b3c9514f96
```
caused by this test chunk:
```lua
prepfile"assert(arg==nil)"
prepfile("assert(arg)", otherprog)
RUN("lua -l%s - < %s", prog, otherprog)
```

LuaJIT v2.1.0-beta3 has extension from Lua 5.3:
The argument table `arg` can be read (and modified) by `LUA_INIT` and `-e` chunks.
See commit 92d9ff211ae864777a8580b5a7326d5f408161ce (Set arg table before evaluating LUA_INIT and -e chunks.).

**Comment by Buristan** [comment]
This test chunk can be adapted from PUC-Rio Lua 5.3 test suite like this:
```diff
@@ -72,9 +93,10 @@ RUN('lua "-e " -- %s a b c', prog)
 -- See commit 92d9ff211ae864777a8580b5a7326d5f408161ce
 -- (Set arg table before evaluating LUA_INIT and -e chunks.).
 -- See also https://github.com/tarantool/tarantool/issues/5686
--- prepfile"assert(arg==nil)"
--- prepfile("assert(arg)", otherprog)
--- RUN("lua -l%s - < %s", prog, otherprog)
+-- Test is adapted from PUC-Rio Lua 5.3 test suite.
+prepfile"assert(arg)"
+prepfile("assert(arg)", otherprog)
+RUN('env LUA_PATH="?;;" lua -l%s - < %s', prog, otherprog)
 
 prepfile""
 RUN("lua - < %s > %s", prog, out)
```

**Comment by igormunkin** [comment]
The issue is resolved in scope of tarantool/luajit@6127f81. The submodule was updated in tarantool in 2.8.0-249-g8a09e18ce (8a09e18), 2.7.1-210-g5f4298b6f (5f4298b), 2.6.2-203-g8343a8129 (8343a81), 1.10.9-119-g7a4c7d0b0 (7a4c7d0).
